packages/provider/src/wallet.ts

```diff
@@ -717,6 +717,10 @@ export const DefaultProviderConfig: ProviderConfig = {
 
 let walletInstance: Wallet | undefined
 
+export const configWallet = (network?: string | number, config?: Partial<ProviderConfig>) => {
+  walletInstance = new Wallet(network, config)
```

Comment:
This isn't safe - if you call this more than once, you'll break sequence, since it'll set up the transports more than once ever.

Reply:
this is the same as we have right now which is `const wallet = new sequence.Wallet(network, config)` .. and people survive. Dev's should know you don't config all the time, just at init time. In fact, in many cases it won't be called, but for advanced usage with singleton, we need a way to recreate the singleton when necessary

Reply:
right, we're trying to work around this problem with this singleton stuff
it's never safe to create more than one wallet instance
there's no "recreating the singleton" that works with the current code setup, this method does not function as intended.